docs/source/pre-requisite.md

```diff
@@ -18,7 +20,7 @@ If you don't have a GitHub token, you can create one following GitHub docs: [cre
 
 ### OpenShift Environment
 
-Using [Operate First][1] you just need a Google account to login to [JupyterHub][2]. Using Operate First is encouraged.
+If you are using Meteor, it will build a [JupyterHub][2] environment for your project from the GitHub repository URL you provide to it. Outside of Meteor, using [Operate First][1] you just need a Google account to login to [JupyterHub][2]. Using Operate First is encouraged.
```

Comment:
```suggestion
If you are using Meteor, it will build a [JupyterHub][2] environment for your project from the GitHub repository URL you provide to it. Otherwise, we recommend using [Operate First][1], where you will just need a Google account to login to [JupyterHub][2].
```

Reply:
changes made :)